src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -99,22 +99,56 @@ interface Player {
 /**
  * @public
  */
-interface RunningScript {
+export interface RunningScript {
```

Comment:
Added docs to the NS-facing `RunningScript` object, pulled from the internal `RunningScript` type